tests/defaults.yml

```diff
@@ -1,7 +1,5 @@
 parameters:
   thanos:
-    commonConfig:
-      version: v0.23.0
```

Comment:
Why is this removed?

Reply:
I thought I'd use the default, which is still v0.18